components/playground/main.go

```diff
@@ -147,19 +155,40 @@ func hasDashboard(pdAddr string) bool {
 	return false
 }
 
-func bootCluster(version string, pdNum, tidbNum, tikvNum int, host string, monitor bool) error {
+func getAbsolutePath(binPath string) string {
+	if !strings.HasPrefix(binPath, "/") {
```

Comment:
I think we shouldn't join with `localdata.EnvNameWorkDir` if the user specifies the directory start with `~`, it is relative to home directory.